tests/services/application.service.spec.ts

```diff
@@ -1,18 +1,38 @@
 import { ApplicationService } from '../../src/app/core/services/application.service';
 
-const applicationService = new ApplicationService(null);
 const currentVersion = '3.2.5';
 
-describe('Test the ApplicationService', () => {
-    it('Test whether the ApplicationService can detect outdated versions', () => {
-        expect(applicationService.isOutdatedVersion('2.2.5', currentVersion)).toBe(false);
-        expect(applicationService.isOutdatedVersion('3.1', currentVersion)).toBe(false);
-        expect(applicationService.isOutdatedVersion('3.2.4', currentVersion)).toBe(false);
-        expect(applicationService.isOutdatedVersion('3.2.4.8.9', currentVersion)).toBe(false);
-        expect(applicationService.isOutdatedVersion(currentVersion, currentVersion)).toBe(false);
-        expect(applicationService.isOutdatedVersion('3.2.5.0', currentVersion)).toBe(true);
-        expect(applicationService.isOutdatedVersion('3.2.5.5', currentVersion)).toBe(true);
-        expect(applicationService.isOutdatedVersion('3.2.6', currentVersion)).toBe(true);
-        expect(applicationService.isOutdatedVersion('10', currentVersion)).toBe(true);
-    });
+class ApplicationServiceStub extends ApplicationService {
+  readonly currentVersion: string;
+  constructor(latestVersion: string, currentVersion: string) {
+    super(null);
+    this.latestVersion = latestVersion;
+    this.currentVersion = currentVersion;
+  }
+}
+
+describe('Test for ApplicationService#isApplicationOutdated', () => {
+  it('should return an Observable of false if the ApplicationService is outdated', () => {
+    const outdatedAppService1 = new ApplicationServiceStub('2.2.5', currentVersion);
+    outdatedAppService1.isApplicationOutdated().subscribe((bool) => expect(bool).toBe(false));
+    const outdatedAppService2 = new ApplicationServiceStub('3.1', currentVersion);
+    outdatedAppService2.isApplicationOutdated().subscribe((bool) => expect(bool).toBe(false));
+    const outdatedAppService3 = new ApplicationServiceStub('3.2.4', currentVersion);
+    outdatedAppService3.isApplicationOutdated().subscribe((bool) => expect(bool).toBe(false));
+    const longVersionedAppService = new ApplicationServiceStub('3.2.4.8.9', currentVersion);
+    longVersionedAppService.isApplicationOutdated().subscribe((bool) => expect(bool).toBe(false));
+    const latestApplicationService = new ApplicationServiceStub(currentVersion, currentVersion);
+    latestApplicationService.isApplicationOutdated().subscribe((bool) => expect(bool).toBe(false));
+  });
+
+  it('should return an Observable of true if the ApplicationService is up to date', () => {
```

Comment:
As these cases are quite simple, let's group the positive and negative cases together in a single test (keep things concise).